main.gd

```diff
@@ -2,19 +2,34 @@
 class_name MainScript
 extends Node
 
+@onready var piece_scene = load("res://piece.tscn")
+
+var initial_piece
+
 
 ## Called when the node enters the scene tree for the first time.
 func _ready():
-	## Enables unhandled input (gameplay input) processing by the _unhandled_key_input virtual function.
-	set_process_unhandled_key_input(true)
+	randomize()  ## randomize rng seed
+	initial_piece = piece_scene.instantiate()
+	initial_piece.setType(randi() % 7)  ## randomize piece type
+	initial_piece.position = Vector2(144, 0)  ## place block at "center" top
+	add_child(initial_piece)
 	pass  # Replace with function body.
 
 
 ## Called when an InputEventKey hasn't been consumed by _input or any GUI Control item (i.e., gameplay input).
 func _unhandled_key_input(event: InputEvent):
+	if event.is_action_pressed("spawn_piece_testing"):
```

Comment:
A comment above denoting which key maps to `spawn_piece_testing` would be clutch so that we don't have to pull up the controls. And a short description of what the action does:

```
# Spawns a piece for testing.
# Key: Spacebar
```